File: 문자열을_정수로_바꾸기.cpp
#include <iostream>
using namespace std;

int solution(string s) {
	bool plus = true;
	if (s[0] == '-') {
		s = s.substr(1);
		plus = false;
	}
	else if (s[0] == '+')
		s = s.substr(1);

	int result = 0;
	for (int i = 0; i < s.length(); i++)
		result = result * 10 + (s[i] - '0');
	if (!plus) result = -result;

	return result;
}

File: 문자열을_정수로_바꾸기.py
def solution(s):
    plus = True
    if s[0] == '-':
        s = s[1:]
        plus = False
    elif s[0] == '+':
        s = s[1:]
        
    result = 0
    for i in range(len(s)):
        result = result * 10 + int(s[i])
    if not plus: result = -result
    
    return result
